iocore/net/SSLUtils.cc

```diff
@@ -95,6 +95,8 @@ struct ssl_user_config {
   ssl_user_config() : opt(SSLCertContext::OPT_NONE)
   {
     REC_ReadConfigInt32(session_ticket_enabled, "proxy.config.ssl.server.session_ticket.enable");
+    REC_ReadConfigStringAlloc(ticket_key_filename, "proxy.config.ssl.server.ticket_key.filename");
+    Debug("ssl", "ticket  key filename %s", (const char *)ticket_key_filename);
   }
   int session_ticket_enabled; // ssl_ticket_enabled - session ticket enabled
```

Comment:
I know it's not in your patch, but can you fix the line where we declare ticket_key_filename such that clang-format doesn't wrap the line? I.e. make the comment shorter (or multiple lines). It just looked really odd, so good time to clean that up here :).